bot/utils/persist.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from pathlib import Path
+from shutil import copyfile
+
+DIRECTORY = Path("data")  # directory that has a persistent volume mapped to it
+
+
+def datafile(file_path: Path) -> Path:
```

Comment:
Not sure I'm crazy about this name. Can we consider some alternatives? `persistant_path` or something seems more descriptive.

Reply:
`persist.persistan_path` seems redundant.

Reply:
I've been trying to think of alternative, but I'm not entirely sure I can think of any super amazing approaches, but your concern is reasonable as it's not super descriptive without context and reading the rest of the docstring.

The `persistent_path` suggestion lends the idea that you're just getting the path for the persistent directory, which is actually what `persist.DIRECTORY` is. I'm also really not a fan of the `double.double` reference that is `persist.persistent_path` or similar. I also kinda think it's less accurate than the current one, as this function also copies the base file to the persistent directory if it's not already there. 

`make_file_persistent` is perhaps closer to what the purpose is, but might feel kinda longish.

@MarkKoz would you have any suggestions?

Reply:
I'll give it a shot with some ideas:

* `persist.copy()`
* `persist.make_persistent()`
* `persist.copy_datafile()`
* `persist.add()`